azure-iot-device/tests/common/pipeline/test_operation_flow.py

```diff
@@ -50,11 +45,11 @@ def stage(mocker, arbitrary_exception, arbitrary_base_exception):
     )
 
 
-@pytest.mark.describe("delegate_to_different_op()")
+@pytest.mark.describe("_send_worker_op_down()")
```

Comment:
As methods, these should have the class in the description

`OperationFlow - .send_worker_op_down()`